crates/solidity/outputs/cargo/tests/src/cst_output/generated/string_literal.rs

```diff
@@ -14,6 +14,11 @@ fn escape_newline_single_quote() -> Result<()> {
     run("StringLiteral", "escape_newline_single_quote")
 }
 
+#[test]
+fn escape_newline_single_quote_crlf() -> Result<()> {
+    run("StringLiteral", "escape_newline_single_quote_crlf")
```

Comment:
I suggest adding another test for just `cr` as well, and renaming the existing tests to distinguish between them:

- `escape_lf_single_quote`
- `escape_crlf_single_quote`
- `escape_cr_single_quote`
- etc... for double quote

Reply:
Done in https://github.com/NomicFoundation/slang/pull/879/commits/b2a6ef4c5058f551492dc338eed3f8aaa93fae79